src/System.Diagnostics.TraceSource/tests/DefaultTraceListenerClassTests.cs

```diff
@@ -45,6 +53,16 @@ public void Fail()
         }
 
         [Fact]
+        public void Fail_WithoutWriteLineOverride()
```

Comment:
If you haven't already done so, it'd be good to look at a code coverage report for the library with your changes and see what functionality you've touched isn't covered, then add tests to cover at least those lines/branches.